### DISCUSSION TITLE
Point Vector Tiles Support

### DISCUSSION THREAD

**Discussion by tech-maven-geospatial** [TerriaJS/terriajs#4891] [Q&A] [answered]
Is there plans to add Point Vector Tile Support?
I was reviewing OpenSphere (Cesium/OpenLayers) Implementation of Vector Tiles https://github.com/ngageoint/opensphere
We've buffered Points to create a small polygon and we can get that working but want to save that extra step

**Comment by rowanwins** [comment, marked as answer]
Hi @tech-maven-geospatial 

Are you talking about support for something like a vector tile catalog item that supports point features?

Eg perhaps something like
````
{
   "type": "vector-tiles",
   "name":  "a bunch of points",
   "url": "https://www.mytiles.com/someAbribratryVectorTilesUrl"
}
````

If so having more fully fledged vector tile support is something we've talked about for a little while however it's never made it up the priority list.

There is an [open issue](https://github.com/TerriaJS/terriajs/issues/4633) which is probably worth following.

Cheers

**Comment by tephenavies** [comment]
@tech-maven-geospatial We've been thinking of extending vector support in Terria for a while. [The issue Rowan posted](https://github.com/TerriaJS/terriajs/issues/4633) is the best place to follow updates on that. In the mean time I did some experimenting with how we could support points without a major rework to our current vector tile support. Here's a demo: http://ci.terria.io/master-mvt-points/#https://gist.githubusercontent.com/steve9164/3b21cfb05f2fd6719fab4685aef8818b/raw/548eb4e1695cf1976ba28fddd207d9e50fdf1bfe/point-mvt-on-v7-catalog.json

And here's the code changes I made (from `master`, but the changes would be very similar in `next`) https://github.com/TerriaJS/terriajs/compare/master-mvt-points

**Reply by tephenavies** [reply]
It's a bit rough and I don't think we'll pursue merging this work or extending before January/February 2021.
